prody/database/__init__.py

```diff
@@ -70,31 +70,37 @@
 
 .. _GOA: https://www.ebi.ac.uk/GOA/
 
+Swiss-Prot
+================================
+
+The following classes and functions can be used to search and retrieve data from the Swiss-Prot database:
+
+  * :class:`.SwissProt` - class to handle Swiss-Prot data from Expasy
+  * :func:`.getCurrentRelease` - gets current Swiss-Prot release version
+  * :func:`.downloadRelease` - downloads current Swiss-Prot database files
+  * :func:`.saveRelease` - saves new Swiss-Prot release version
+  * :func:`.updateRelease` - updates Swiss-Prot local database
+  * :func:`.getLocalRelease` - checks local Swiss-Prot release version
+  * :func:`.checkForUpdates` - checks wheather there is newer Swiss-Prot version than current local one
 
+.. _GOA: https://www.ebi.ac.uk/GOA/
 """
 
-__all__ = []
 
-from . import pfam
+from . import cath, dali, goa, pfam, quartataweb, swissprot, uniprot
```

Comment:
please don't reformat like this. It's helpful to be able to see the package imports in blocks like we had